torch/linalg/__init__.py

```diff
@@ -140,3 +140,28 @@
     >>> LA.norm(m[0, :, :]), LA.norm(m[1, :, :])
     (tensor(3.7417), tensor(11.2250))
 """)
+
+tensorinv = _add_docstr(_linalg.linalg_tensorinv, r"""
```

Comment:
Just to be clear, this function does not support batching, right?

It seems like it would be very difficult to make it support batching, too, with its current signature.

Reply:
No, it doesn't support batching. I think we could support batching with an additional keyword argument specifying the batching dimensions, but before doing it here it should be done for `tensordot`.